server/utils/placementApplications/utils.ts

```diff
@@ -0,0 +1,36 @@
+import { UserDetails } from '../../@types/ui'
+import { createQueryString } from '../utils'
+import { TabItem } from '../tasks/listTable'
+import assessPaths from '../../paths/assess'
+import { hasRole } from '../users'
+
+export const assessmentsTabItems = (user: UserDetails, activeTab?: string): Array<TabItem> => {
```

Comment:
Given this util is called `assessmentTabItems` and it is used solely on the `/assessments` page I think it would be clearer if it was in the `/server/utils/assessments` folder